code/modules/power/supermatter/supermatter.dm

```diff
@@ -153,7 +155,19 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		/datum/gas/proto_nitrate = 1,
 		/datum/gas/zauker = 1,
 		/datum/gas/miasma = 0.5,
+		/datum/gas/antinoblium = 1,
 	)
+	///The list of gases that reduce powerloss.
+	var/list/powerloss_inhibitor_gas = list(
+		/datum/gas/carbon_dioxide,
+		/datum/gas/antinoblium,
+	)
+	///The list of gasses that cause the SM to shoot nuclear particles, and their odds to do so in %
+	var/list/nuclear_gas = list(
+		/datum/gas/bz = 30,
+		/datum/gas/antinoblium = 100,
```

Comment:
too high, too high

Reply:
What would be a more acceptable value, 60?

Reply:
Maybe, 30 is pretty high as it is. do you see people putting a lot of this stuff in the sm in the first place?

Reply:
I imagine people would combine it with CO2 so it doesn't meltdown. I'll lower or remove this effect if you think it's unnecessary.

Reply:
is it feasible to pump it in at high rates?

Reply:
This depends on the engine. It would be pretty hard to get enough antinoblium for an engine with 8 scrubbers, but if one chose a higher injection to scrubbing ratio engine, one could get enough antinoblium to make an almost pure antinoblium engine.

Reply:
aight